### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading '@value')

### ISSUE THREAD

**Issue by xpat** [NoelDeMartin/media-kraken#29]
[Please explain what you were trying to do when this error appeared.]
I was tyring to access media.kraken https://noeldemartin.github.io/media-kraken/ from my computer.  Earlier today I accessed it from the phone.  This second time, from the computer, I got a message with an option to migrate (my content or something?) and that if I didn't migrate it would cause unexpected behavior with the app.  I chose to migrate and got the error message below: 

Stack trace:
```TypeError: Cannot read properties of undefined (reading '@value')
    at Proxy.<anonymous> (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:32:35334)
    at l (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:39:176583)
    at Generator._invoke (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:39:177913)
    at Generator.next (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:39:177012)
    at r (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:39:52607)
    at s (https://noeldemartin.github.io/media-kraken/js/0.86b415cd.worker.js?__WB_REVISION__=3c88596dc40416fa8ab7f54da6468da8:39:52810)
```

**Comment by NoelDeMartin** [comment]
Hey @xpat, thanks for opening an issue.

Looking at the error, it seems like it's failing trying to migrate the container metadata, but this is a problem I detected a couple of weeks ago with [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer) and [I disabled it for now](https://github.com/NoelDeMartin/media-kraken/commit/36ffa342a344314ceff67f1a12e15620353ff464). So it shouldn't be happening with the latest version of the app. Can you make sure that you're using version 0.1.13 or newer?

Also, can you tell me which pod provider you're using? When I enable this back again I'll have to see how it works for different providers.

**Comment by xpat** [comment]
@NoelDeMartin I'm using version 0.1.13, according to the page info from your site.  I'm using my own server (CSS) at https://mypod.dev.  Let me know what else I can do to help out.  I not so familiar with CSS.  I haven't done anything with it except set up a server and as far as app development goes, I've done nothing.  I am pretty impressed with your app. It's only one of three or four that I've used and it looks pretty sleek and has some nice functionality and looks good.

**Comment by NoelDeMartin** [comment]
Oops, sorry about that. It seems like the fixes I mentioned weren't included in production yet, I added them after releasing 0.1.13. I've released a new 0.1.14 version and that one should work. As I mentioned, I found this error with CSS as well, so I'm pretty sure it should be fixed with the new version.

That explains why you were getting the error, so I'm closing this issue for now. Let me know if something else is broken or we should reopen it.

Also, thanks for the nice words 😄️. If you like this app, maybe you also like the one I've been working on recently. I'm about to release it soon :). It's called [Umai](https://github.com/noeldemartin/umai).
